README.md

```diff
@@ -114,6 +119,11 @@ All the parameters mentioned below are **mandatory**, so don't forget to provide
     <td>Python interpreter which will be used for running the tests.</td>
     <td>python3</td>
   </tr>
+  <tr>
+    <td>RUNNING_WAIT</td>
+    <td>Count of seconds that tests will be wait for bot running <b>(docker container only)</b>.</td>
```

Comment:
* ...will be waiting...
* ...for the bot running...
* (for Docker container only)